File: VB/CustomPrintPreview/Grid/GetGridPreviewEventArgs.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Columns

Namespace CustomPrintPreview.Grid
	Public Class GetGridPreviewEventArgs
		Inherits EventArgs
		Private fGridPreview As GridControl
		Public Property GridPreview() As GridControl
			Get
				Return fGridPreview
			End Get
			Set(ByVal value As GridControl)
				fGridPreview = value
			End Set
		End Property

		Private fGroupColumn As GridColumn
		Public Property GroupColumn() As GridColumn
			Get
				Return fGroupColumn
			End Get
			Set(ByVal value As GridColumn)
				fGroupColumn = value
			End Set
		End Property
	End Class
End Namespace

File: CS/CustomPrintPreview/Grid/GetGridPreviewEventArgs.cs
using System;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Columns;

namespace CustomPrintPreview.Grid {
    public class GetGridPreviewEventArgs :EventArgs {
        GridControl fGridPreview;
        public GridControl GridPreview {
            get { return fGridPreview; }
            set { fGridPreview = value; }
        }

        GridColumn fGroupColumn;
        public GridColumn GroupColumn {
            get { return fGroupColumn; }
            set { fGroupColumn = value; }
        }
    }
}
